src/store/actions/workStyle.js

```diff
@@ -1,9 +1,29 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
+  if (data) {
+    const { title_primary, title_complementary, Cards } = data;
+    const sanitizedCards = Cards.map(card => {
+      return {
+        description: card.description,
+        Title: card.Title,
+        image_main: { url: card.image_main.url },
+        image_main_hover: { url: card.image_main_hover.url },
+        color: card.color
+      };
```

Comment:
I think the code will be more readable if we use ES6 restructuring. 
and we can use the Aliasing if two extracted data has the same name.
Ex : 
```
({description, objectProperty: anotherName  })=>{
     return {
               title,
               descriptions,
               anotherName
             }
    }
```
Then you need to change how to consume the data in the component. `WorkStyleCard`